registry/registry.go

```diff
@@ -200,6 +209,44 @@ func (registry *Registry) ListenAndServe() error {
 	return registry.server.Serve(ln)
 }
 
+// configureLetsEncryptChallenge adds a listener on port 80 for a "http-01" challenge.
+// This is required because Let's Encrypt disabled TLS-SNI-01 and TLS-SNI-02 challenges on 09.01.2018.
+// See: https://community.letsencrypt.org/t/important-what-you-need-to-know-about-tls-sni-validation-issues/50811
+func (registry *Registry) configureLetsEncryptChallenge(manager *autocert.Manager, port int) {
+	s := &http.Server{
+		Handler: manager.HTTPHandler(nil),
```

Comment:
What are the requirements for this handler? (Feel free to just point me at the section in the documentation if it is not a simple answer)

Reply:
```
If HTTPHandler is never called, the Manager will only use TLS SNI challenges for domain verification. 
```

at https://godoc.org/golang.org/x/crypto/acme/autocert#Manager.HTTPHandler

An example, scroll down to example at https://godoc.org/golang.org/x/crypto/acme/autocert#Manager .